packages/eslint-config-base/index.js

```diff
@@ -23,6 +23,30 @@ module.exports = {
       'always',
       { exceptAfterSingleLine: true }
     ],
-    'import/no-extraneous-dependencies': 'error'
+    'import/no-extraneous-dependencies': 'error',
+    'sort-imports': [
+      'error',
+      {
+        ignoreCase: true,
+        ignoreDeclarationSort: true
+      }
+    ],
+    'import/order': [
+      'error',
+      {
+        groups: ['builtin', 'external', 'internal', 'type'],
+        pathGroups: [
+          {
+            pattern: '@/**',
+            group: 'internal'
+          }
```

Comment:
I like to keep `vue` and `vitest` at the top. What do you think? Should we add separate group for it?

@wa0x6e

Reply:
Good idea. I tend to always put the main package at the top

Reply:
added a fix for this